the-collab-lab/smart-shopping-list-deprecated
# Move project from yarn to npm

## Description

We've used `deyarn` to move this template project from `yarn` to `npm`. 

This decision was made because we experienced participants encountering some unnecessary friction in getting yarn set up on their machines and we'd like to avoid as much of those troubles as possible going forward.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓   | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓    | :link: Update dependencies |
|   ✓  | :scroll: Docs              |

## Related Issue
To get more details on the conversation, see [this Slack conversation](https://the-collab-lab.slack.com/archives/GLQ89ULE6/p1576808365001200).

## Before / After

* **Before:** project used `yarn` as package manager
* **After:** project uses `npm` as package manager

## Testing Steps / QA Criteria

* Pull updates locally, run `npm install` to update dependencies
* Ensure you can launch the project successfully with `npm start`
